github/resource_github_repository_pull_request.go

```diff
@@ -289,13 +289,13 @@ func resourceGithubRepositoryPullRequestUpdate(d *schema.ResourceData, meta any)
 		return resourceGithubRepositoryPullRequestRead(d, meta)
 	}
 
-	errs := []string{fmt.Sprintf("could not update the Pull Request: %v", err)}
+	errors := []string{fmt.Sprintf("could not update the Pull Request: %v", err)}
```

Comment:
```suggestion
	errs := []string{fmt.Sprintf("could not update the Pull Request: %v", err)}
```

Why did you change this? This would shadow the `errors` package in the stdlib and even if it's not currently being used in this file it's not idiomatic.